### DISCUSSION TITLE
Distinguish between mapset owner and guest mapper in song select screen

### DISCUSSION THREAD

**Discussion by sineplusx** [ppy/osu#19962] [Ideas] [answered]
This discussion has already been brought up in issue [#12965](https://github.com/ppy/osu/issues/12965), but I didn't want to write it in that thread since I have no fix for the actual "issue".

My idea is basically what @LeNitrous already suggested: The lazer client should make a distinction between the user who owns the mapset (hosts a collaboration) and the participants of a collab which map individual difficulties. Right now the aforementioned issue is still present - clicking on the link for the mapset owner sends you to the guest mappers userpage, depending on what difficulty is selected.

The way I would change that is by replacing the names on the difficulty panels of the song select screen (mapped by ... ) with the names of the guest mappers, and adding another line with a userpage link on the overview similar to the already existing one.

Example using https://osu.ppy.sh/beatmapsets/423527

Currently
![Current](https://user-images.githubusercontent.com/112003827/186611484-8a777d7e-e488-43cb-96f7-71ed297f0976.png)

Proposal
![Proposal](https://user-images.githubusercontent.com/112003827/186611661-96adbb01-3863-4749-b79e-6dfb780570f0.png)

I'd really like to hear your thoughts on this.

**Comment by frenzibyte** [comment, marked as answer]
Showing both the host and the guest one below the other looks bad. The new design of the beatmap info overlay achieves this pretty nicely by separating between the overall beatmap metadata section and the difficulty section:

<img width="1072" alt="CleanShot 2022-08-25 at 12 32 23@2x" src="https://user-images.githubusercontent.com/22781491/186629917-457c12ee-75f9-4d16-b8ed-28589298bfa2.png">

And the same can potentially be applied in the new design of the beatmap select wedge:

<img width="1000" alt="CleanShot 2022-08-25 at 12 33 08@2x" src="https://user-images.githubusercontent.com/22781491/186630042-9320d9a1-47ac-4c0b-a2e2-bed343e45e45.png">

But I can't see this working well with the current design. The best that can be done is mentioning the host of the beatmap in the details section at the same place where "Description" is placed, but not sure that'll look any better. @peppy thoughts?

**Comment by peppy** [comment]
I think what you have there is fine, but no need to show the host ever. Just show the guest mapper.
